### ISSUE TITLE
22/Autoencoder

### ISSUE THREAD

**Issue by wonjae124** [wonjae124/2021_Lab_project#11]

**Comment by wonjae124** [comment]
[  복원 결과  ]
1.
epoch 50회시 코스트 : Training Loss: 0.004499	 validation Loss 0.006014
![image](https://user-images.githubusercontent.com/67944072/162614601-c8e64749-6bbc-43e5-a4ea-2383106a8c33.png)

2.  객관적 화질 분석 지표
![image](https://user-images.githubusercontent.com/67944072/162614672-9c29dc36-0da5-447a-989c-4be56271e3a8.png)
\
![image](https://user-images.githubusercontent.com/67944072/162614629-8f104921-e0a1-420e-b616-acbbe4e7f781.png)
